components/RecipeForm.js

```diff
@@ -45,98 +46,137 @@ export default function RecipeForm({ onSubmit }) {
     onSubmit(newData);
   }
   return (
-    <form onSubmit={handleSubmit}>
-      <StyledTop>
+    <Fragment>
+      <StyledTop $height={imageUrl}>
         <IconButton
+          right="1rem"
+          top="1rem"
           style={"x"}
           onClick={() => {
             router.back();
           }}
         ></IconButton>
+        {imageUrl && (
+          <StyledImageCloudinary
+            src={imageUrl}
+            alt="Uploaded Image"
+            width={100}
+            height={300}
+          />
+        )}
+        <StyledImageUploadContainer>
+          <Plus width={40} height={40} />
+          <StyledImageUpload type="file" onChange={onChange} />
+        </StyledImageUploadContainer>
       </StyledTop>
-      <StyledArticle>
-        <StyledBigInput
-          type="text"
-          name="title"
-          placeholder="Titel"
-          required
-        ></StyledBigInput>
-        <StyledListItem>
-          <StyledInput
-            type="number"
-            name="duration"
-            placeholder="Dauer"
-            $width={"5rem"}
+      <form onSubmit={handleSubmit}>
+        <StyledArticle>
+          <StyledBigInput
+            type="text"
+            name="title"
+            placeholder="Titel"
             required
-            min="0"
-          ></StyledInput>
-          <StyledP>min</StyledP>
+          ></StyledBigInput>
+          <StyledListItem>
+            <StyledInput
+              type="number"
+              name="duration"
+              placeholder="Dauer"
+              $width={"5rem"}
+              required
+              min="0"
+            ></StyledInput>
+            <StyledP>min</StyledP>
 
-          <StyledDropDown
-            onChange={(e) => setDifficulty(e.target.value)}
-            value={difficulty}
-            name="difficulty"
+            <StyledDropDown
+              onChange={(e) => setDifficulty(e.target.value)}
+              value={difficulty}
+              name="difficulty"
+              required
+            >
+              <option value="easy">Anfänger</option>
+              <option value="medium">Fortgeschritten</option>
+              <option value="hard">Profi</option>
+            </StyledDropDown>
+          </StyledListItem>
+          <StyledH2>Zutaten</StyledH2>
+          <StyledList>
+            {ingredients.map((ingredient, index) => (
+              <StyledListItem key={index}>
+                <StyledInput
+                  value={ingredient.quantity}
+                  onChange={(e) => handleInputChange(e, index, "quantity")}
```

Comment:
please be explicit when naming your parameters and also apply this for other instances

```suggestion
                  onChange={(event) => handleInputChange(event, index, "quantity")}
```